/mnt/qb/work/butz/svolz67/envs/lib/python3.10/site-packages/numpy/core/fromnumeric.py:3440: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/mnt/qb/work/butz/svolz67/envs/lib/python3.10/site-packages/numpy/core/_methods.py:189: RuntimeWarning: invalid value encountered in double_scalars
  ret = ret.dtype.type(ret / rcount)
Traceback (most recent call last):
  File "/mnt/qb/work/butz/svolz67/dev/models/master/./runBatch.py", line 340, in <module>
    output = seq(input_images, horizon)
  File "/mnt/qb/work/butz/svolz67/envs/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1110, in _call_impl
    return forward_call(*input, **kwargs)
  File "/mnt/qb/work/butz/svolz67/dev/models/master/./runBatch.py", line 113, in forward
    h[i], c[i] = layer(z, h[i], c[i])
  File "/mnt/qb/work/butz/svolz67/envs/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1110, in _call_impl
    return forward_call(*input, **kwargs)
  File "/mnt/qb/work/butz/svolz67/dev/models/master/models.py", line 13, in forward
    c = sigmoid(f) * c + sigmoid(i) + tanh(g)
RuntimeError: CUDA out of memory. Tried to allocate 10.00 MiB (GPU 0; 10.76 GiB total capacity; 8.65 GiB already allocated; 2.69 MiB free; 9.85 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
Traceback (most recent call last):
  File "/mnt/qb/work/butz/svolz67/dev/models/master/./runBatch.py", line 340, in <module>
    output = seq(input_images, horizon)
  File "/mnt/qb/work/butz/svolz67/envs/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1110, in _call_impl
    return forward_call(*input, **kwargs)
  File "/mnt/qb/work/butz/svolz67/dev/models/master/./runBatch.py", line 113, in forward
    h[i], c[i] = layer(z, h[i], c[i])
  File "/mnt/qb/work/butz/svolz67/envs/lib/python3.10/site-packages/torch/nn/modules/module.py", line 1110, in _call_impl
    return forward_call(*input, **kwargs)
  File "/mnt/qb/work/butz/svolz67/dev/models/master/models.py", line 13, in forward
    c = sigmoid(f) * c + sigmoid(i) + tanh(g)
RuntimeError: CUDA out of memory. Tried to allocate 10.00 MiB (GPU 0; 10.76 GiB total capacity; 8.65 GiB already allocated; 2.69 MiB free; 9.85 GiB reserved in total by PyTorch) If reserved memory is >> allocated memory try setting max_split_size_mb to avoid fragmentation.  See documentation for Memory Management and PYTORCH_CUDA_ALLOC_CONF
